config.yaml

```diff
@@ -70,6 +65,11 @@ demo: &demo
               log:
                 severity: Debug+
 
+  witness:
+    db:
+      path: "/var/db/dscp-witness"
```

Comment:
This needs to be overridden for production, so not much point in putting it here.